v0.26: Importing dumps into instances with existing DBs returns an error (#1469)
meilisearch/documentation: learn/advanced/dumps.md

If a dump file is visible in the file system, the dump process was successfully completed. **Meilisearch will never create a partial dump file**, even if you interrupt an instance while it is generating a dump.

::: note

> Unlike [tasks](/learn/advanced/asynchronous_operations.md), dumps have no queue. **Meilisearch only processes one dump at a time.** If you attempt to create a dump while another dump is still processing, Meilisearch will throw an [error](/errors). The task queue will not process any further tasks during dump creation, but you can still add new requests to the queue. This is also true for [snapshots](/learn/advanced/snapshots.md#snapshots).

Reviewer: So much talk of queues can be confusing. I think it's more clear with a paragraph break, but that's just my opinion.

```suggestion
Unlike [tasks](/learn/advanced/asynchronous_operations.md), dumps have no queue. **Meilisearch only processes one dump at a time.** If you attempt to create a dump while another dump is still processing, Meilisearch will throw an [error](/errors).

The task queue will not process any further tasks during dump creation, but you can still add new requests to the task queue. This is also true for [snapshots](/learn/advanced/snapshots.md#snapshots).
```

> Unlike [tasks](/learn/advanced/asynchronous_operations.md), dumps have no queue. **Meilisearch only processes one dump at a time.** If you attempt to create a dump while another dump is still processing, Meilisearch will throw an [error](/errors). The task queue can receive other future operations to perform but will not process any further tasks during dump creation. It will still accept new tasks in the queue. This is also true for [snapshots](/learn/advanced/snapshots.md#snapshots).

Reviewer 2: I'm not sure the second new sentence is necessary—unless I misunderstand it, we are just repeating what we said before, no?

I also propose we invert the order of the information: first we say Meilisearch will not process tasks, then we say it will still queue them. What do you think?

```suggestion
Unlike [tasks](/learn/advanced/asynchronous_operations.md), dumps have no queue. **Meilisearch only processes one dump at a time.** If you attempt to create a dump while another dump is still processing, Meilisearch will throw an [error](/errors). The task queue will not process any further tasks during dump creation, but you can still add new requests to the queue. This is also true for [snapshots](/learn/advanced/snapshots.md#snapshots).
```

## Importing a dump

> When a dump is being imported, the API is not available to the task queue. As a result, no read or write operations can be performed, except importing the dump into the database at that time.

Reviewer 2: ```suggestion
When a dump is being imported, the API is not available to the task queue. As a result, no read or write operations can be performed until the importing process is complete.
```
